Documentation/fonts/swift2.md

```diff
@@ -19,6 +19,7 @@ You can customize some elements of this template by overriding the following par
 | Parameter Name | Default Value | Description |
 | -------------- | ------------- | ----------- |
 | `enumName` | `FontFamily` | Allows you to change the name of the generated `enum` containing all font families. |
+| `preservePath` | N/A | Setting this parameter will disable the basename filter applied to all font paths. Use this if you are adding fonts to your target in a specific folder, and need to preserve those folder names when registering fonts. The path will be relative to the folder you provided to SwiftGen. |
```

Comment:
Use this if you added your font folder as a "folder reference" in your Xcode project, making that folder hierarchy preserved once copied in the build app bundle